corehq/apps/commtrack/util.py

```diff
@@ -149,36 +148,22 @@ def _needs_commtrack_config(domain):
     make_product(domain.name, 'Sample Product 2', 'pq', program.get_id)
     make_product(domain.name, 'Sample Product 3', 'pr', program.get_id)
 
-    domain.location_types = [
-        LocationType(
-            name='state',
-            allowed_parents=[''],
-            administrative=True
-        ),
-        LocationType(
-            name='district',
-            allowed_parents=['state'],
-            administrative=True
-        ),
-        LocationType(
-            name='block',
-            allowed_parents=['district'],
-            administrative=True
-        ),
-        LocationType(
-            name='village',
-            allowed_parents=['block'],
-            administrative=True
-        ),
-        LocationType(
-            name='outlet',
-            allowed_parents=['village']
-        ),
-    ]
-    # this method is called during domain's post save, so this
-    # is a little tricky, but it happens after the config is
-    # created so should not cause problems
-    domain.save()
+    previous = None
+    for name, administrative in [
+        ('state', True),
+        ('district', True),
+        ('block', True),
+        ('village', True),
+        ('outlet', False),
+    ]:
+        location_type = LocationType(
+            domain=domain.name,
+            name=name,
+            parent_type=previous,
+            administrative=administrative,
+        )
+        location_type.save()
+        previous = location_type
```

Comment:
Ahhh.. I might actually be removing this after recent feedback from Sheel that we shouldn't populate bogus types.